app/views/dashboards/show.html.erb

```diff
@@ -12,17 +12,19 @@
     <% end %>
   </p>
 
-  <table class="table">
-    <thead>
-    <tr>
-      <th></th>
-      <% @environment.deploy_groups.sort_by(&:natural_order).each do |deploy_group| %>
-        <th><%= link_to deploy_group.name, deploy_group %></th>
-      <% end %>
-    </tr>
-    </thead>
-    <tbody>
-      <tr ng-repeat="project in projects" class="{{ project.css.tr_class }}" style="{{ project.css.style }}" project_deploy_groups/>
-    </tbody>
-  </table>
+  <div class="large-table-responsive">
+    <table class="table">
```

Comment:
This is needed for the better fix, header that stays on screen and first column that stays in view.